.changeset/config.json

```diff
@@ -10,12 +10,13 @@
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
   "ignore": [
+    "amplify-ui-angular-mono",
+    "angular-example",
     "docs",
+    "e2e",
     "environments",
-    "angular-example",
     "next-example",
-    "vue-example",
-    "amplify-ui-angular-mono",
-    "e2e"
+    "react-native-example",
```

Comment:
Alphabetized while I was here, but the change here is just adding "react-native-example".